code/code/disc/disc_defense.cc

```diff
@@ -6,16 +6,20 @@
 
 
 #include "disc_defense.h"
+#include "being.h"
+
 
 CDDefense::CDDefense() :
   CDiscipline(),
-  skAdvancedDefense()
+  skAdvancedDefense(),
+  skFocusedAvoidance()
```

Comment:
Chances are that these constructors, destructors, operator= don't do anything useful and should be deleted.